File: src/main/java/com/comeup/function/Person.java
package com.comeup.function;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

/**
 * @author: qiu wanzi
 * @date: 2024年3月7日 0007
 * @version: 1.0
 * @description: TODO
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Person {
    private String name;
    private int age;
    private String sex;
    private String phone;
    private String email;
    private Date createTime;
    private String idCard;
    private String bankCard;
    private String bankName;
    private String bankAccount;
    private String bankAddress;
    private String bankPhone;
    private String bankEmail;
}

File: src/main/kotlin/com/comeup/kotlin/Person.kt
package com.comeup.kotlin

/**
 * 功能描述:
 * @author: qiu wanzi
 * @date: 2024年3月28日 0028
 * @version: 1.0
 */
class Person (id: Int, name: String, age: Int) {
    var name = name;
    var id = id;
    var age = age;

    override fun toString(): String {
        return "Person(id=$id, name='$name', age=$age)"
    }
}